android-components/components/feature/addons/src/main/java/mozilla/components/feature/addons/Addon.kt

```diff
@@ -287,6 +298,10 @@ data class Addon(
             val permissions = extension.getMetadata()?.permissions.orEmpty() +
                 extension.getMetadata()?.hostPermissions.orEmpty()
 
+            val averageRating = extension.getMetadata()?.averageRating ?: 0f
```

Comment:
I am wondering: could we rewrite this method to call `getMetadata()` once? Maybe just with:

```
val metadata = extension.getMetadata()
val description = metadata?.description ?: extension.id
val permissions = metadata?.permissions.orEmpty() + metadata?.hostPermissions.orEmpty()
val averageRating = metadata?.averageRating ?: 0f

[...]
```